<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageViewerTest</class>
 <widget class="QWidget" name="ImageViewerTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>图片浏览</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ImageViewer" name="imageViewer" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="sizeTypeComboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="fadeCkb">
       <property name="text">
        <string>逐渐显示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="keyMoveCkb">
       <property name="text">
        <string>按键切换</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openBtn">
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QWidget</extends>
   <header>imageviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
